index.ts

```diff
@@ -115,6 +247,29 @@ export class Scenarios {
     });
   }
 
+  /**
+   * Adds more {@link CallbackMutateProject} callbacks to all derived scenarios. 
```

Comment:
Maybe something like: "Derives new scenarios from each of the previously defined scenarios"

Reply:
Done.